pysts/vocab.py

```diff
@@ -56,7 +56,12 @@ def embmatrix(self, emb):
             try:
                 embedding_weights[index, :] = emb.g[word]
             except KeyError:
-                embedding_weights[index, :] = np.random.uniform(-0.25, 0.25, emb.N)  # 0.25 is embedding SD
+                if index == 0:
+                    embedding_weights[index, :] = np.zeros(emb.N)
+                else:
+                    embedding_weights[index, :] = np.random.uniform(-0.25, 0.25, emb.N)  # 0.25 is embedding SD
+
+
```

Comment:
Adventurous change!  However, it'll impact all other models too.  I have started a sanity check evaluation on yoda dataset with just this change to confirm if we'll need to rebenchmark everything again or it's just a small change.